app/serializers/v2/report_serializer.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module V2
+  # JSON serialization for Reports
+  class ReportSerializer < V2::ApplicationSerializer
+    attributes :title, :description, :business_objective, :compliance_threshold
+
+    derived_attribute :os_major_version, security_guide: [:os_major_version]
+    derived_attribute :profile_title, profile: [:title]
+    derived_attribute :ref_id, profile: [:ref_id]
+
+    aggregated_attribute :system_count, :assigned_systems, V2::Report::SYSTEM_COUNT
+    aggregated_attribute :compliant_system_count, :systems, V2::Report::COMPLIANT_SYSTEM_COUNT
+    aggregated_attribute :unsupported_system_count, :systems, V2::Report::UNSUPPORTED_SYSTEM_COUNT
+    aggregated_attribute :test_result_system_count, :systems, V2::Report::SYSTEM_COUNT
+  end
```

Comment:
Don't forget about that additional boolean attribute that tells if the report has all the assigned systems exposed or not. There's a `total_system_count` field on the table for this purpose.